experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+    eventName = "showModalFlow",
+    flowName = params.$Input.flowName,
+    model = skuid.$M(params.$Input.model);
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: flowName,
+        params: model.getRows()
+    };
+
+// If there is a matching global application event with this name,
+// then fire that event, with each Skuid Event parameter
+// being mapped directly to that Event parameter.
+if (auraEvt) {
+    if (paramsObject) {
+        auraEvt.setParams(paramsObject);
+    }
+    auraEvt.fire();
+}
+
+skuidAuraEvt.setParams({
+    "name": eventName,
+    "source": "skuid",
+});
+
+// ALSO fire a generic "Skuid Event",
+// where all of the parameters are serialized to JSON
+if (paramsObject) {
+    skuidAuraEvt.setParam("data", paramsObject);
+}
+
+skuidAuraEvt.fire();</jsitem>
+			<jsitem location="inlinesnippet" name="showVariable" cachelocation="false">const params = arguments[0];
+const eventName = "showModalFlow";
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: "Civilian_Training_Form",
+        params:[{name:"cid",type:"String",value:params.row.Id}]
+    };
+
+// If there is a matching global application event with this name,
+// then fire that event, with each Skuid Event parameter
+// being mapped directly to that Event parameter.
+if (auraEvt) {
+    if (paramsObject) {
+        auraEvt.setParams(paramsObject);
+    }
+    auraEvt.fire();
+}
+
+skuidAuraEvt.setParams({
+    "name": eventName,
+    "source": "skuid",
+});
+
+// ALSO fire a generic "Skuid Event",
+// where all of the parameters are serialized to JSON
+if (paramsObject) {
+    skuidAuraEvt.setParam("data", paramsObject);
+}
+
+skuidAuraEvt.fire();</jsitem>
+		</javascript>
+		<css/>
+		<actionsequences>
+			<actionsequence id="5c35d805-abbb-4f15-bfc0-e02b9d68268d" label="test" type="event-triggered" event-scope="global" event-name="civilianTrainingFlowModalEvent">
+				<description/>
+				<actions>
+					<action type="showToast" duration="4" position="bl" message="woohooo!&#10;{{$Input.cid}}" enabled="false"/>
+				</actions>
+				<inputs>
+					<input type="value" name="cid"/>
+				</inputs>
+			</actionsequence>
+			<actionsequence id="29f52636-5af5-49b1-8550-54170e17e336" label="CallFlow" type="reusable">
+				<description/>
+				<actions>
+					<action type="custom" snippet="callFlow"/>
+				</actions>
+				<inputs>
+					<input type="value" name="flowName"/>
+					<input type="model" name="model"/>
+				</inputs>
+			</actionsequence>
+			<actionsequence id="bc602b89-5c69-4b6e-aaee-8012a420a264" label="TestFlow" type="reusable">
```

Comment:
This doesn't appear to be called from anywhere in the example. Is it needed or is it leftover from debugging while building?